### PR TITLE
Consider nonprofit timezone when displaying dates

### PR DESCRIPTION
**Pull request by clarissalimab**
Closes [tix 2750](https://github.com/CommitChange/tix/issues/2750).

### CODE REVIEW

File: spec/lib/query/query_supporters_spec.rb [hunk 1]
```diff
@@ -10,14 +10,15 @@
 
   
   let(:np) { force_create(:nonprofit)}
-  let(:supporter1) { force_create(:supporter, nonprofit: np)}
+  let(:supporter1) { force_create(:supporter, nonprofit: np, name: 'Cacau')}
```
Comment (wwahammy) [inline spec/lib/query/query_supporters_spec.rb]: :smile: :dog:

Comment (clarissalimab) [reply spec/lib/query/query_supporters_spec.rb]: it was necessary :laughing:

**Comment by wwahammy** [comment]
This PR is only addressing payment timing issues with searching supporters, not the payments dashboard, right?

**Review by wwahammy — CHANGES_REQUESTED** [review]
Searchwise, this does fix the referenced issue. However, it doesn't address the issue of the last payment listed on the dashboard being always in UTC. That should be addressed as well to close this issue.

If you feel it can be done this sprint, we can move it into here. Otherwise, we can split it out. What do you think @clarissalimab?

Example of the problem:
Screenshot from 2021-05-12 12-00-38


But look at the time for the recurring donation on 3/31 at: 
Screenshot from 2021-05-12 12-00-27

**Comment by clarissalimab** [comment]
@wwahammy I'll take a look!

**Comment by clarissalimab** [comment]
@wwahammy, it's working now with the fix we did yesterday!

This is how it looks on the supporters list:
image

This is how the details look like (I've updated my local database, that's why there are more registers there):
image

**Review by wwahammy — APPROVED** [review]
I think this all looks good. Let's ship it!

**Comment by wwahammy** [comment]
So, I just thought of something: are dates on exports being properly exported in the nonprofit's timezone? We should check that although I think that might be a separate issue.

**Comment by clarissalimab** [comment]
@wwahammy I'll check that, if not, I'll create an issue, okay?

**Pull request merged**